var/spack/repos/builtin/packages/optipng/package.py

```diff
@@ -18,3 +18,4 @@ class Optipng(AutotoolsPackage, SourceforgePackage):
     sourceforge_mirror_path = "optipng/optipng-0.7.7.tar.gz"
 
     version('0.7.7', sha256='4f32f233cef870b3f95d3ad6428bfe4224ef34908f1b42b0badf858216654452')
+    patch('for_aarch64.patch', when='target=aarch64:')
```

Comment:
Can you please put a reference to https://github.com/imagemin/optipng-bin/issues/97 in a comment above the patch?

Reply:
I added a comment.